website/docs/r/service_aws.html.markdown

```diff
@@ -12,6 +12,16 @@ This resource allows you to manage a service account that is tied to an AWS IAM
 
 The native AWS identity provider uses a combination of AWS IAM and AWS KMS to provide access to SecretHub for any service running on AWS (e.g. EC2, Lambda or ECS) without needing a SecretHub credential.
 
+## Example Usage
+
+```terraform
+resource "secrethub_service_aws" "demo_service_account" {
+  repo        = "workspace/repo"
+  role        = "${aws_iam_role.your_application.name}"
+  kms_key_arn = "${aws_kms_key.secrethub_auth.arn}"
```

Comment:
The KMS key is not for authentication, it's for encryption. So, do we want to call it `secrethub_auth`?

Reply:
Yes, it's the KMS key that facilitates the SecretHub authentication.

Reply:
Yes, it facilitates authentication to the SecretHub server and e2e encryption for the secrets.

I find the latter just as important (if not more important) than the former. Can we come up with a name that reflects both purposes of the KMS encryption key?

Reply:
But KMS doesn't do secrets decryption, it just bootstraps the app upon launch, so `secrethub_bootstrap`? Or we can say okay that's just an implementation detail, let's call it `secrethub_e2e`.

Reply:
Yeah, `secrethub_e2e` works for me :+1: